keps/sig-storage/20191125-bucket-provisioning.md

```diff
@@ -338,3 +339,170 @@ parameters: string:string [8]
 1. `secretRef`: (Optional) The name and namespace of an existing secret to be copied to the `Bucket`'s namespace for static provisioning.  Requires that `bucketIdentifier` be defined. Used for brownfield and static cases.
 1. `parameters`: (Optional) Object store specific key-value pairs passed to the driver.
 
+### RPC
+
+```
+syntax = "proto3";
+
+package cosi.v1;
+
+import "google/protobuf/descriptor.proto";
+
+option go_package = "github.com/container-object-store-interface/go-cosi";
+
+extend google.protobuf.MessageOptions {
+    bool cosi_secret = 1000;
+}
+
+message DriverInfoRequest {} // INTENTIONALLY BLANK
+
+enum DataProtocol {
+    PROTOCOL_UNSPECIFIED = 0;
+    AZURE = 1;
```

Comment:
Do we want consistency in this name here vs. in the API fields? Eg. should this be named `AZUREBLOB` or should the API protocol value be "azure"?

Reply:
I think you're right, will fix.